README.md

```diff
@@ -11,4 +11,55 @@ Supported formats:
    ```
     gradlew clean build
    ``` 
-3. Find the distribution in build/distributions.
\ No newline at end of file
+3. Find the distribution in build/distributions.
+
+## Transform Parasoft Reports
+
+### Transform Parasoft Static Analysis XML Report to SARIF Report
+1. Run XMLToSARIF script with the generated XML report:
+   * For Windows:
+      ```cmd
+      path\to\XMLToSARIF.bat -i <path\to\Parasoft\XML\report> [-o <path\to\output\SARIF\report>] [-t <path\to\parasoft\tool\installation\or\Java\home\directory>]
+      ```
+   * For Linux:
+      ```shell
+      path/to/XMLToSARIF.sh -i <path/to/Parasoft/XML/report> [-o <path/to/output/SARIF/report>] [-t <path/to/parasoft/tool/installation/or/Java/home/directory>]
+      ```
+2. If a path is provided to the `-o` option, the SARIF report should be generated in the specified path. If not, the SARIF report will be generated in the same directory as input XML report.
+
+### Parameters in command line:
+| Parameter                          | Description                                                                                                                                  |
+|------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------|
+| -i, --inputXmlReport (required)    | The path to Parasoft XML report that you want to transform to SARIF report.                                                                  |
+| -o, --outputSarifReport (Optional) | The path to output SARIF report. The default is the same path of inputed XML report.                                                         |
+| -t, --toolOrJavaHomeDir (Optional) | The path to Parasoft tools installation or Java home directory. The default is Java home directory path imported from environment variables. |
```

Comment:
I think `The default is Java home directory path imported from environment variables.` is not suitable. We can say `If not specified, transformer will try to use Java from system environment.`